azure-toolkit-libs/azure-toolkit-auth-lib/src/main/java/com/microsoft/azure/toolkit/lib/auth/util/AzureEnvironmentUtils.java

```diff
@@ -69,6 +74,20 @@ public static AzureEnvironment stringToAzureEnvironment(String environment) {
             .findFirst().orElse(null);
     }
 
+    public static String getPortalUrl(AzureEnvironment env) {
+        if (AzureEnvironment.AZURE.equals(env)) {
+            return GLOBAL_PORTAL;
+        } else if (AzureEnvironment.AZURE_CHINA.equals(env)) {
+            return CHINA_PORTAL;
+        } else if (AzureEnvironment.AZURE_GERMANY.equals(env)) {
+            return com.microsoft.azure.AzureEnvironment.AZURE_GERMANY.portal();
+        } else if (AzureEnvironment.AZURE_US_GOVERNMENT.equals(env)) {
+            return com.microsoft.azure.AzureEnvironment.AZURE_US_GOVERNMENT.portal();
```

Comment:
this url is not right :        put("portalUrl", "https://manage.windowsazure.us");